pkg/rootless/rootless_linux.c

```diff
@@ -94,28 +110,63 @@ reexec_userns_join (int userns)
 
   argv = get_cmd_line_args (ppid);
   if (argv == NULL)
-    _exit (EXIT_FAILURE);
+    {
+      fprintf (stderr, "cannot read argv: %s\n", strerror (errno));
+      _exit (EXIT_FAILURE);
+    }
 
   pid = fork ();
+  if (pid < 0)
+    fprintf (stderr, "cannot fork: %s\n", strerror (errno));
```

Comment:
I'm not sure, should we error out if the fork is less than 0 (creating a child process failed)?  I think it's OK as is, but wasn't 100% sure.

Reply:
we error out a few lines below, where we return for `pid != 0` so it covers both the error condition and the successful fork.

Reply:
Maybe I'm lacking tea, but I'm not seeing a "pid != 0" anywhere in the file...